### PR TITLE
Update mitsubishi2mqtt.ino (fixed widevane error in compilation)

### PR DESCRIPTION
**Pull request by mamrai1**
1.fixed widevane error in compilation
2. removed dublicate entry of widevane mode 6

### CODE REVIEW

**Comment by melyux** [comment]
Looks good! Fixes the typo. Disregard my push to my branch with the same thing.

**Review by melyux — CHANGES_REQUESTED** [review]
@mamrai1 In line 1288, `const size_t bufferSizeInfo = JSON_OBJECT_SIZE(7);` needs to be changed to 8 instead of 7, since we added the wideVane key. Otherwise I've found that it cuts off the last key (in this case, "mode").

**Comment by gysmo38** [comment]
Thank you

File: src/mitsubishi2mqtt/mitsubishi2mqtt.ino [hunk 1]
```diff
@@ -1408,7 +1408,16 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
     hp.update();
   }
   else if (strcmp(topic, ha_temp_set_topic.c_str()) == 0) {
+    //add to fix HP turn off after change temperature
+    heatpumpSettings currentSettings = hp.getSettings();
+    delay(10);
```
Comment (melyux) [inline src/mitsubishi2mqtt/mitsubishi2mqtt.ino:1413]: Was there a reason for this naked 10 second delay? Seems kind of drastic!

**Comment by mamrai1** [comment]
Maybe 7 sec would be enough?

File: src/mitsubishi2mqtt/mitsubishi2mqtt.ino (hunk 1 above)
Comment (lesleypersyn) [reply src/mitsubishi2mqtt/mitsubishi2mqtt.ino:1413]: Syntax
delay(ms)
ms: the number of milliseconds to pause

Comment (melyux) [reply src/mitsubishi2mqtt/mitsubishi2mqtt.ino:1413]: Doh. I think he was going for 10 seconds though

**Comment by melyux** [comment]
But what is the purpose of the delay?

**Comment by melyux** [comment]
That delay doesn't seem to fix the turning off issue. Mine turns off anyway by itself sometimes.

**Pull request merged**